### PR TITLE
Small fixes for the demo.

### PR DESCRIPTION
**Pull request by nsthorat**
Add datasets:
- lmsys
- openorca
- oasst2

Other:
- Reduce spacy spam
- Add OpenAI JSON format as a dataset format so we can cluster role=human lmsys.

### CODE REVIEW

File: lilac/formats/default_formats.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 """Registers all available dataset formats."""
 
+from lilac.formats.openai_json import OpenAIJSON
```
Comment (brilee) [inline lilac/formats/default_formats.py:3]: lilac in import

File: lilac/formats/openai_json.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+def _openai_selector(item: Item, conv_role: str) -> str:
+  """Selector for OpenAI JSON Formatted conversations."""
+  # TODO(nsthorat): Make this return an array, and not pre-join with newlines.
+  values = [conv['content'] for conv in item['conversation'] if conv['role'] == conv_role]
+  # Get the __value__ key version of text if it's enriched.
+  values = [value.get(VALUE_KEY, value) for value in values]
+  return '\n'.join(values)
+
+
+_USER_SELECTOR = DatasetFormatInputSelector(
```
Comment (brilee) [inline lilac/formats/openai_json.py:18]: may as well have a system selector as well?

**Review by brilee — APPROVED** [review]

File: lilac/formats/openai_json.py (hunk 2 above)
Comment (nsthorat) [reply lilac/formats/openai_json.py:18]: done

File: lilac/formats/default_formats.py (hunk 1 above)
Comment (nsthorat) [reply lilac/formats/default_formats.py]: done

**Review by dsmilkov — APPROVED** [review]

**Pull request merged**